src/components/bds-switch/test/bds-switch.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { BdsSwitch } from '../bds-switch';
+
+describe('bds-switch', () => {
```

Comment:
you should add unit tests that cover all the properties, methods, states, styling, etc